### PR TITLE
Update pixiv privacy policy selector

### PR DESCRIPTION
**Pull request by afisher3578**

### CODE REVIEW

File: declarations/pixiv.json [hunk 1]
```diff
@@ -4,7 +4,10 @@
     "Privacy Policy": {
       "fetch": "https://policies.pixiv.net/en.html#privacy",
```
Comment (MattiSG) [inline declarations/pixiv.json:5]: This now redirects, let's store the redirect target 🙂 
```suggestion
      "fetch": "https://policies.pixiv.net/en/privacy_policy.html",
```

File: declarations/pixiv.json [hunk 2]
```diff
@@ -4,7 +4,10 @@
...
       "select": [
-        "#privacy"
+        "main"
```
Comment (MattiSG) [inline declarations/pixiv.json:7]: > I'd personally find it more stable with a more clearly named selector.
```suggestion
        ".terms-block"
```

File: declarations/pixiv.json [hunk 3]
```diff
@@ -4,7 +4,10 @@
...
+      ],
+      "remove": [
+        "img"
```
Comment (MattiSG) [inline declarations/pixiv.json:10]: I don't find any image on https://policies.pixiv.net/en/privacy_policy.html, where did you find one? 🙂

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: declarations/pixiv.json (hunk 2 above)
Comment (afisher3578) [reply declarations/pixiv.json:7]: Agreed, thanks for pointing that out.

File: declarations/pixiv.json (hunk 3 above)
Comment (afisher3578) [reply declarations/pixiv.json:10]: That is correct now looking into it. There are no images. When I ran the ota engine locally and in the web ui the "Master Terms of Use" were displayed which had images. 
Screenshot 2024-06-27 at 9 09 03 AM
Screenshot 2024-06-27 at 9 09 50 AM

**Review by MattiSG — APPROVED** [review]
Looks perfect, thanks! ✨

**Pull request merged**